src/llvm-simdloop.cpp

```diff
@@ -172,14 +172,18 @@ void LowerSIMDLoop::enableUnsafeAlgebraIfReduction(PHINode *Phi, Loop *L) const
 
 bool LowerSIMDLoop::runOnModule(Module &M)
 {
-    Function *simdloop_marker = M.getFunction("julia.simdloop_marker");
-
-    if (!simdloop_marker)
-        return false;
+    bool ivdep = false;
+    Function *marker = M.getFunction("julia.simdloop_marker");
+    if (!marker) {
+        ivdep = true;
+        marker = M.getFunction("julia.simdivdep_marker");
+        if (!marker)
+            return false;
+    }
```

Comment:
So you only get to use one of these options per Module? :P

Reply:
Oh that's what this is doing :P I thought that at first but then I convinced myself otherwise.
So I should look at `marker->users()`... I didn't want to duplicate that code.

Reply:
>  I didn't want to duplicate that code.

If only there was a way to write code once and then run it with two different values...